service/controller/clusterapi/v29/drainer_resource_set.go

```diff
@@ -106,25 +104,10 @@ func NewDrainerResourceSet(config DrainerResourceSetConfig) (*controller.Resourc
 		}
 	}
 
-	var tccpOutputsResource controller.Resource
-	{
-		c := tccpoutputs.Config{
-			Logger: config.Logger,
-
-			Route53Enabled: config.Route53Enabled,
-		}
-
-		tccpOutputsResource, err = tccpoutputs.New(c)
-		if err != nil {
-			return nil, microerror.Mask(err)
-		}
-	}
-
 	resources := []controller.Resource{
 		awsClientResource,
 		machineDeploymentResource,
-		tccpOutputsResource,
-		asgStatusResource,
+		//asgStatusResource,
```

Comment:
Preparing this for later. I cannot activate this as this would break the controller as it still watches `Cluster` CRs. The drainer controller is essentially broken anyway and does nothing with node pool clusters right now. I move over the drainer resources in another PR and activate the resources together later once it is good to go.